docs/examples/sheet/defaultExample.js

```diff
@@ -180,7 +27,160 @@ export default function AccessibilityExample(): Node {
       </Box>
       {shouldShow && (
         <Layer zIndex={sheetZIndex}>
-          <SheetWithAccessibilityLabels onDismiss={() => setShouldShow(false)} />
+          <Sheet
+            accessibilityDismissButtonLabel="Close audience creation sheet"
+            accessibilitySheetLabel="Audience list creation for new campaign"
+            heading="Create a new audience list"
+            onDismiss={() => {}}
+            // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
This can be solved by adding `useCallback`

Reply:
done